# This message is used to send diagnostic information about the state of the robot
Header header #for timestamp

RosSingularity[] singularities